src/resources/js/tickets-block.js

```diff
@@ -45,16 +45,48 @@ tribe.tickets.block = {
 		'.tribe-block__tickets__item__quantity__remove, .tribe-block__tickets__item__quantity__add',
 		function( e ) {
 			e.preventDefault();
-			var input  = $( this ).parent().find( 'input[type="number"]' );
-			var add    = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var input = $( this ).parent().find( 'input[type="number"]' );
+			var add = $( this ).hasClass( 'tribe-block__tickets__item__quantity__add' );
+			var step = ( undefined !== Number( input[ 0 ].step ) ) ? Number( input[ 0 ].step ) : 1;
```

Comment:
do we support step for the input field, and are we using it?

Reply:
https://github.com/moderntribe/event-tickets/blob/master/src/views/blocks/tickets/quantity-number.php#L28

Reply:
gotcha, instead of checking specifically for `undefined` here we can do the following instead: `var step = input[ 0 ].step ? Number( input [ 0 ].step ) : 1` to catch any other cases that might be falsey.